my-app/src/Person/Person.js

```diff
@@ -1,7 +1,15 @@
 import React from "react";
+import './Person.css'
 
 const Person = (props) => {
-  return <p>I'm {props.name}..
-  i am {props.age} Years Old </p> 
+  return (
+    <div className="Person">
```

Comment:
give class name in named convention like person, person-container (in lowercase)